src/app/core/services/localeService.ts

```diff
@@ -1,24 +1,27 @@
 import { Injectable } from '@angular/core';
-import { appConfig } from 'src/app/shared/helpers/config';
+import { EMPTY, Observable, of } from 'rxjs';
+import { appConfig, LanguageConfig } from 'src/app/shared/helpers/config';
+
 
 @Injectable({
   providedIn: 'root'
 })
 export class LocaleService {
 
-  readonly languages?: { tag: string, path: string }[];
-  readonly currentTag?: string;
+  readonly languages: LanguageConfig[];
+  readonly currentLang: LanguageConfig | undefined;
```

Comment:
`currentLang` cannot be defined-only by design because even if we display a fatal error, the app is still displayed in the background. In that use case only the `currentLang` will be `undefined`, but that case is also handled

Reply:
Ok.
So write it:
```ts
readonly currentLang?: LanguageConfig;
```

Reply:
Fixed